jenkins-scripts/lib/homebrew_bottle_creation.bash

```diff
@@ -32,21 +53,22 @@ echo '# BEGIN SECTION: run test-bot'
 # The test-bot makes a full cleanup of all installed pkgs. Be sure of install back
 # mercurial to keep the slave working
 export HOMEBREW_DEVELOPER=1
-brew tap homebrew/test-bot
-git -C $(brew --repo)/Library/Taps/homebrew/homebrew-test-bot \
-    fetch ${TEST_BOT_REPO} ${TEST_BOT_BRANCH}
-git -C $(brew --repo)/Library/Taps/homebrew/homebrew-test-bot \
-    checkout FETCH_HEAD
+brew tap osrf/simulation
+# replace with 'hub -C $(brew --repo osrf/simulation) pr checkout ${ghprbPullId}'
+# after the following hub issue is resolved:
+# https://github.com/github/hub/issues/2612
+pushd $(brew --repo osrf/simulation) && \
+  hub pr checkout ${ghprbPullId} && \
+  popd
 
-git -C $(brew --repo) fetch ${BREW_REPO} ${BREW_BRANCH}
-git -C $(brew --repo) checkout FETCH_HEAD
+# test-bot wants to 'git fetch --unshallow' over ssh, which has permission issues
+# explicitly unshallow using https instead
+git -C $(brew --repo osrf/simulation) fetch --unshallow \
+    https://github.com/${GITHUB_REPOSITORY}
 
 brew test-bot --tap=osrf/simulation \
               --fail-fast \
-              --root-url=https://osrf-distributions.s3.amazonaws.com/bottles-simulation \
-              --ci-pr ${PULL_REQUEST_URL} \
-            || { brew install hg; exit -1; }
```

Comment:
should we assume that git is not provided by brew in our Mac nodes? or should we leave `brew install git` to avoid possible problems with test-bot leaving systems without git?

Reply:
git is provided by the Xcode command-line tools, so we can assume it is there